main/observing/observation_run.py

```diff
@@ -15,14 +17,27 @@ def __init__(self, observation_request_list, image_directory):
         '''
         self.image_directory = image_directory
         self.observation_request_list = observation_request_list
-        self.filterwheel_dict = filter_wheel.get_filter().filter_position_dict()
         self.camera = Camera()
-
+        #self.telescope = Telescope()
+        
+        self.filterwheel_dict = {value: int(key.replace('position_', '')) for key,value in
+                                 ObjectReader(Reader(r'c:\users\gmu observtory1\-omegalambda\config\fw_config.json')).ticket.__dict__.items()}
```

Comment:
This path should not be hard coded, it should be passed via the driver.  Eventually we will start using a config file for these arguments.